controllers/ironicneutronagent_controller.go

```diff
@@ -647,11 +648,22 @@ func (r *IronicNeutronAgentReconciler) generateServiceConfigMaps(
 
 	cmLabels := labels.GetLabels(instance, labels.GetGroupLabel(ironic.ServiceName), map[string]string{})
 
+	db, err := mariadbv1.GetDatabaseByName(ctx, h, ironic.DatabaseName)
```

Comment:
@hjensas do the neutronagent even need/has access to the db?

Reply:
No direct DB access in the agent.

Reply:
ack, thanks. I have reverted the change to the agent. please have a look when you get to it. right now there is an issue in ci.